File: frontend/static/script/models/population/Woodcutter.js
"use strict";
import { Worker } from "./Worker.js";



export class Woodcutter extends Worker {
    constructor(population_id, name, profession, alive, employed, field_of_work, working, production) {
        super(population_id, name, profession, alive, employed, field_of_work, working, production);
        
    }
}

File: backend_refactoring/CSharpApi/Models/BluePrints/Beings/WoodCutter.cs
namespace CSharpApi.Models.BluePrints.Beings
{
    public class WoodCutter : Worker
    {
        public WoodCutter(string workerName, bool employed = true, string profession = "worker", bool alive = true, string fieldOfWork = "woodcutter", bool working = false, int production = 500) : base(workerName, employed, profession, alive, fieldOfWork, working, production)
        {
        }
    }
}
